packages/rocketchat-statistics/server/functions/get.js

```diff
@@ -32,10 +32,12 @@ RocketChat.statistics.get = function _getStatistics() {
 		}
 	});
 
-	if (statistics.wizard.allowMarketingEmails) {
```

Comment:
any reason to remove this condition? I'm not sure the results will be the expected

Reply:
basically the issue here is if they chose to register we need their email sent.    Otherwise they can't actually register.

Reply:
swapping this out for the setting that indicates registerServer

Reply:
Ah this is true...

Reply:
not really.. actually you can still chose to send statistics (register server) but opt out for newsletter, that's why we have this, to not send any contact information if opted out.

Reply:
You can logically do it, but according to the wizard setup and the wording of the setup wizard that is not how it is supposed to work.

Reply:
idk we're talking about the same thing, because you have a very specific option to opt out from receiving newsletter: 
![image](https://user-images.githubusercontent.com/8591547/50351061-263eee80-0528-11e9-9f3e-481ff965ca98.png)

so if we (rocket.chat) receive this fields (`contactName` and `contactEmail`) we _will_ send offers and newsletters.

Reply:
right.. so sending `allowMarketingEmails` we can read it on our side to determine sending or not :+1: